create_graphics.py

```diff
@@ -468,7 +468,7 @@ def parallel_maps(cla, fhr, ds, level, model, spec, variable, workdir,
     dm.draw(show=True)
 
     # Build the output path
-    png_suffix = level if level != 'ua' else ''
+    png_suffix = level
     png_file = f'{variable}_{tile}_{png_suffix}_f{fhr:03d}.png'
```

Comment:
Since we no longer need the logic, `png_suffix` is no longer needed, really. Would you mind putting `level` directly into the template for the `png_file` definition. Unless, of course, I missed where `png_suffix` is used elsewhere.